src/pages/LendPage/LendDetailPage.tsx

```diff
@@ -0,0 +1,450 @@
+  },
+}));
+
+const LendDetailPage: React.FC = () => {
+  const classes = useStyles();
+  return (
+    <Box width={'100%'}>
+      <Box
+        display={'flex'}
+        justifyContent={'flex-start'}
+        alignItems={'center'}
+        gridGap={'20px'}
+        flexWrap={'wrap'}
+      >
+        <svg
+          data-name='Layer 2'
+          xmlns='http://www.w3.org/2000/svg'
+          width='28'
+          height='28'
+          viewBox='0 0 28 28'
+        >
+          <g data-name='invisible box'>
+            <path data-name='Rectangle 20001' fill='none' d='M0 0h28v28H0z' />
+          </g>
+          <g data-name='Q3 icons'>
+            <path
+              data-name='Path 11780'
+              d='m16.285 10.35-6.942 7a1.108 1.108 0 0 0 0 1.633l6.942 7a1.225 1.225 0 0 0 1.575.117 1.108 1.108 0 0 0 .117-1.75l-5.017-5.016h12.367a1.167 1.167 0 1 0 0-2.333H12.96l5.017-5.017a1.108 1.108 0 0 0-.117-1.75 1.225 1.225 0 0 0-1.575.117z'
+              transform='translate(-3.744 -4.167)'
+              fill='#448aff'
+            />
+          </g>
+        </svg>
+        <Box fontSize={'24px'} fontWeight={'bold'} color={'white'}>
+          Quickswap Pool
+        </Box>
+        <Box display={'flex'} gridGap={'2px'}>
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <Box
+            bgcolor={'#32394c'}
+            width={'24px'}
+            height={'24px'}
+            borderRadius={'100%'}
+            display={'flex'}
+            alignItems={'center'}
+          >
+            <Box ml={'1px'} mt={'-5px'} display={'flex'}>
+              <USDTIcon size={'13px'} />
+            </Box>
+            <Box ml={'-4px'} mt={'5px'} display={'flex'}>
+              <USDTIcon size={'13px'} />
+            </Box>
+          </Box>
+        </Box>
+      </Box>
+      <Box
+        mt={'40px'}
+        fontSize={'20px'}
+        fontWeight={'700'}
+        textAlign={'left'}
+        color={'#ebecf2'}
+      >
+        Leverage assets you believe in
+      </Box>
+      <Box mt={'23px'} display={'flex'} gridGap={'24px'} flexWrap={'wrap'}>
+        <Box
+          flex={'1'}
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Supply
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+        <Box
+          flex={'1'}
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Borrowed
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M

```

Comment:
Why is 19.2m hardcoded?

Reply:
updated the value on top of zeplin mocup.